_site/index.html

```diff
@@ -559,6 +559,69 @@ <h2 class="subtitle">Here we publish the most interesting things we experienced
   </div>
 </section>
 
+
+<section class="section section-dark contact-us">
+    
+    <div class="content">
+        <div class="text-center">
+            <h1 class="transite transite-up sn-1">Let's get connected to build an excellent application</h1>
+            <h2 id="contact-form-subtitle" class="transite transite-up sn-2">Leave your contact details, and we will reach you within 24 hours</h2>
+        </div>
+        <form id="contact-form" class="form-wrapper transite transite-up sn-3">
+            <div class="input-wrapper">
+                <input id="contact-form-name-input" name="name" placeholder="Your name" type="text">
+                <input id="contact-form-email-input" name="email" placeholder="Your E-mail" type="text">
+            </div>
+            <textarea id="contact-form-comment-input" name="comment" placeholder="Your text" rows="10"></textarea>
+            <div class="file-wrapper">
+                <label id="contact-form-custom-btn" class="file-label" for="contact-form-file-upload">Choose
+                    file</label>
+                <input id="contact-form-file-upload" name="file" type="file" hidden="hidden">
+                <div id="contact-form-custom-text" class="text-helper"></div>
+                <div id="contact-form-remove-file" class="remove-btn">
```

Comment:
You can have `class="remove-btn hidden"` from the start. Then you don't need to add `display: none;` in styles.
You have `hidden` class for that purpose. Just add or remove it.

Please re-check all the places where you are handling elements visibility. Do not add `display: smth` in JS, toggle `hidden` class